torch/csrc/api/include/torch/nn/functional/loss.h

```diff
@@ -17,6 +17,20 @@ inline Tensor hinge_embedding_loss(
       options.reduction());
 }
 
+inline Tensor multi_margin_loss(
+    const Tensor& input,
+    const Tensor& target,
+    const MultiMarginLossOptions& options = {}) {
```

Comment:
It would be great to add the following checks (expressed in Python) to match the Python version better:
```python
    if p != 1 and p != 2:
        raise ValueError('only p == 1 and p == 2 supported')
    if weight is not None:
        if weight.dim() != 1:
            raise ValueError('weight must be one-dimensional')
```